src/Components/Links.js

```diff
@@ -24,7 +20,7 @@ function Links({ links }) {
               className="p-p-3 p-m-2 p-button-outlined"
               style={{ color: colors[link.icon] }}
               key={`link.url_${index}`}
-              onClick={() => goToLinkHandle(link.url)}
+              onClick={() => window.open(link.url)}
```

Comment:
I think you need to do this way 
`onClick={()=> window.open(link.url,"_blank")}`